parity-clib/parity.h

```diff
@@ -17,6 +17,11 @@
 #ifndef _PARITY_H_INCLUDED_
 #define _PARITY_H_INCLUDED_
 
+#define		PARITY_CALLBACK_RESTART			0
```

Comment:
I think you misunderstood my concern. We don't need this. All we need is an extra parameter that the user can pass to `parity_rpc` and `parity_subscribe_ws` and that is transparently passed to the callback.

Reply:
Ok, one more dumb question from my side then!

How do you propose to safely cast the ```void *``` to some object then? 
Or are we assuming that the user will always pass the same type as `user_data` such as an user-defined struct?

Reply:
It is the user's job to interpret this `void*` the way they want, we only guarantee to pass back the same value. But yeah normally it is pointing to a user-defined structure.